src/interactive-window/interactiveWindowCommandListener.ts

```diff
@@ -216,21 +218,21 @@ export class InteractiveWindowCommandListener implements IDataScienceCommandList
                                     directoryChange = uri;
                                 }
 
-                                const notebook = await this.jupyterExporter.translateToNotebook(
+                                const notebook = await this.jupyterExporter?.translateToNotebook(
```

Comment:
Could we check the existance of this earlier? Seems like we're doing work (showing the dialog and stuff) when we might not actually have the ability to export.

Reply:
sure, though I still need this check for typescript to be happy